charts/substra-backend/templates/job-migrations.yaml

```diff
@@ -45,15 +45,21 @@ spec:
             done < /accounts/users
 
             ## IncomingOrganization
-            while read -r user_password; do
-                read user password <<< "$user_password"
-                ./manage.py create_incoming_organization "$user" "$password"
+            while IFS= read -r org_password; do
+              # Extract the password as the last word
+              password=$(echo "$user_password" | awk '{print $NF}')
+              # Extract the username by removing the last word
+              orgname=$(echo "$user_password" | sed 's/ [^ ]*$//')
+              ./manage.py create_incoming_organization "$orgname" "$password"
             done < /accounts/incoming_organizations
 
             ## OutgoingOrganization
-            while read -r user_password; do
-                read user password <<< "$user_password"
-                ./manage.py create_outgoing_organization "$user" "$password"
+            while IFS= read -r org_password; do
+              # Extract the password as the last word
+              password=$(echo "$org_password" | awk '{print $NF}')
+              # Extract the username by removing the last word
+              orgname=$(echo "$org_password" | sed 's/ [^ ]*$//')
+              ./manage.py create_outgoing_organization "$orgname" "$password"
```

Comment:
Is there a specific reason we are using bash and not refactoring that in some django command ? 

I can see we are using bash to read a file, and then calling a django command with the parsed content. Wouldn't make sense to create a command reading the file and parsing it itself ?

Reply:
To be honest I did not challenge the bash implementation that was already in place because:
* I needed to move fast on this
* I needed the code to be available through chart publication without having to release an applicative image

Reply:
I'm personally okay with merging as is, and improve it as a follow-up

Reply:
https://linear.app/owkin/issue/FL-1581/improve-django-cli-for-users-and-organization-management